index.html

```diff
@@ -1,12 +1,144 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>code review</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Media Player Project</title>    
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="css/style.css">
+    
 </head>
 <body>
-  
+    <section>
+        <nav>
+            <div class="menu-item">
+                <span><i class="fa fa-bars"></i></span>
+                <div id="start">
+                    <img class="yt-logo" src="img/YouTube-Logo.wine.png" alt="">
+                </div>
+            </div>
+            <div id="center">
+                <form class="yt-search-form" action="">
+                    <div class="yt-search">
+                        <input type="text" placeholder="Search...">
+                        <button type="submit"><i class="fa fa-search"></i></button>
+                    </div>
+                    <span class="mic-icon"><i class="fa fa-microphone"></i></span>
+                </form>
+            </div>
+            <div id="end">
+                <span><i class="ico fa fa-video"></i></span>
+                <span><i class="ico fa fa-bell"></i></span>
+                <span><i class="ico fa fa-user"></i></span>
+            </div>
```

Comment:
Great work, but I think if you can use list (ul) instead, it will be semantic.

Reply:
Nice one. :+1: